Open-Code.md

```diff
@@ -0,0 +1,36 @@
+# Open code in the NHS
+
+## Introduction
+
+There are number of organisations and publications advocating open coding in its various guises within the NHS and more generally across the public sector and academia.  This page is intended as a place to collate pertinent literature and proposed/tried approaches to implementation of open code policies in situations in the NHS from individual teams up to entire organisations.
+
+There is also a significant crossover between open code and software engineering best practice, which means that they are often (always?) promoted together as an effective means to improve reproducibility, 
+
+### Glossary (DRAFT)
+
+- **Reproducible Analytics Pipelines**: analytics processes developed in open source programming languages and adhering to software engineering best practices to allow for reproducing analyses with very little effort.
+- **Repo/Repository**: a set of files organised in a project for a specific purpose (such as [statements-on-tools](https://github.com/nhs-r-community/statements-on-tools) itself), containing code or documentation under version control.
+
+## References
+
+The following is a list of some of the material available online discussing and supporting open coding and software engineering approaches to code development in the NHS:
+
+- [The Goldacre Report](https://www.gov.uk/government/publications/better-broader-safer-using-health-data-for-research-and-analysis) - a systematic and far-reaching report, written on behalf of the Department of Health, advocating for open coding and reproducible analytical pipelines (RAP) in the NHS,
+- [Office for Statistics Regulation – Overcoming Barriers to Adoption of RAP](https://osr.statisticsauthority.gov.uk/publication/reproducible-analytical-pipelines-overcoming-barriers-to-adoption/) - a report written in support of RAP for adoption by all government departments doing analytics, describing the challenges and recommending solutions to address those challenges covering both organisational, team-level and individual barriers,
+- [NHSX Open Source Policy](https://github.com/nhsx/open-source-policy) (currently in DRAFT) - a comprehensive description of why and how open source should eb implemented in the NHS, including statements about best practice and a checklist for open sourcing code,
```

Comment:
typo: eb -> be

Reply:
I'm new to this: have you made this change?  So if I click "Resolve conversation" will it be changed?

Reply:
I haven't made any changes to the PR - just left comments :-)

Reply:
Done the change